recipes/mender-shell/debian-master/control

```diff
@@ -0,0 +1,13 @@
+Source: mender-shell
+Maintainer: Mender Team <mender@northern.tech>
+Section: misc
+Priority: optional
+Standards-Version: 3.9.2
+Build-Depends: debhelper (>= 9)
+
+Package: mender-shell
+Conflicts: mender
```

Comment:
Question:
So the debian package is still `mender` ?

Reply:
This should not be here, a leftover from mender-client copy paste

But there is correct, as "mender-client" package conflicts with old "mender" package.